other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,128 @@
+
+Logins can be enforced by [requiring user logins](https://docs.altis-dxp.com/security/require-login/), forcing [HTTP basic authentication](https://docs.altis-dxp.com/security/php-basic-auth/), or both.
+
+In your Altis configuration file (`composer.json`), add the following to both require login and HTTP Basic authentication. You may omit Basic authentication, but we recommend having at least one of these enabled for all non-production and pre-launch sites. You can override the require login setting to not require login on local environments. Basic authentication is disabled by default on local environments. Refer to the documentation pages for more information about Require Login and Basic Authentication.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"security": {
+					"require-login": true
+				}
+			},
+			"environments": {
+				"local": {
+					"modules": {
+						"security": {
+							"require-login": false
+						}
+					}
+				}
+			}
+		}
+	}
+}
+```
+
+## Preparation for launch
+
+After the site launch is completed, there may be some items left to clean up or check.
+
+- Remove the `require-login` and/or the `php-basic-auth` requirements on the production site. They should remain active on any staging or development environments.
+- Check the Search Console, Analytics or other similar services or logs for any errors after the deploy is complete.
+- Do the development or staging environments need to be synced?
+
+## Site launch
+
+Before the domain(s) are pointed to the Altis servers, a content migration should be performed to ensure the right content is loaded on the site when the switch is made active. When the most updated data has been imported into the database, a search-and-replace action should be performed so all the database entries that contain the URL of the site have been updated to the _correct_ URL. This can be done in the Altis Dashboard by either opening an SSH connection to the Sandbox Server or running a command on the WP CLI tab.
+
+The following command would change all entries in the database containing `domain-production.altis.cloud` to `domain.com`. We recommend testing the change first using the `--dry-run` flag to make sure no unexpected tables are affected.
+
+```bash
+wp search-replace domain-production.altis.cloud domain.com --all-tables --network --url=domain-production.altis.cloud
+```
+
+This step will need to be repeated for any subdomains you have. Be sure to flush the cache when you are done, otherwise the old URLs will still be saved in the object cache.
+
+```bash
+wp cache flush
+```
+
+After the database is updated and the site is ready to go, contact Altis Support to let them know that you are ready for the DNS switch to your new site.
```

Comment:
Anything else here? Do we need to specify contacting Altis Support via their launch ticket/Altis Dashboard vs Slack?

Reply:
I think using the launch ticket makes sense to track the whole process and act as a log we can show. It makes sense to add that as a suggestion.

Reply:
@roborourke Is that not sufficiently covered [here](https://github.com/humanmade/altis-documentation/pull/222/files#diff-6cf7fa890a1e26dae62794924a67bdaf3f4bd7fee7a8f72479a6f4a4539e5ac6R23)?

Reply:
Yeah true, staring at it too long!